cmd/soroban-cli/src/commands/events.rs

```diff
@@ -67,7 +66,11 @@ pub struct Cmd {
     /// Though the specification supports multiple filter objects (i.e.
     /// combinations of type, IDs, and topics), only one set can be specified on
     /// the command-line today, though that set can have multiple IDs/topics.
```

Comment:
Seems like this change means we support multiple now, so we could update the comment.

Reply:
I think the behavior remains unchanged.

Prev: `multiple = true, max_values(5)`
New: `num_args = 1..=5`

Looking at the comment on `topic` below, I think this comment is trying to tell us that if you pass a combination of `--id`, `--topic`, and `--type`, they will all be combined with AND, rather than OR. I did not change the filtering logic, so I believe this should still be true.

Changing the logic or even the comment seems outside the scope of this PR.

Reply:
Oh yeah. You're right. That comment is somewhat confusingly worded...